cypress/e2e/course/deactivationOfCtlTools.feature

```diff
@@ -0,0 +1,116 @@
+Feature: Deactivation of ctl tools
+
+  @stable_test
+  Scenario: As an admin I can see deactivated external tool in external tool section.
+    Given I am logged in as a 'admin1_nbc' at 'nbc'
+    When I go to administration page
+    When I go to school administration
+    When I go to new school administration page
+    When I click on external tools panel
+    Then I see the external tools table
+    Then I can see 'CY Test Tool deactivated External Tool' is deactivated in tool table
+
+    @stable_test
+  Scenario: As an admin I can add a tool and deactivate it for my school.
+    Given I am logged in as a 'admin1_nbc' at 'nbc'
+    When I go to administration page
+    When I go to school administration
+    When I go to new school administration page
+    When I click on external tools panel
+    When I click the add external tool button
+      Then I can select the tool 'CY Test Tool active External Tool' from available tools
+      Then I can deactivate the tool
+      Then I can fill required parameter
```

Comment:
I think it could be useful to parameterize this

Reply:
parameterization is not possible here, because the datatestid is generated in relation to the key of the parameter. We need the exact key which is not displayed in the ui, to use it as input for cypress selector ..